VCFv4.3.tex

```diff
@@ -577,43 +591,113 @@ \section{INFO keys used for structural variants}
-##INFO=<ID=DGVID,Number=1,Type=String,Description="ID of this element in Database of Genomic Variation">
-##INFO=<ID=DBVARID,Number=1,Type=String,Description="ID of this element in DBVAR">
-##INFO=<ID=DBRIPID,Number=1,Type=String,Description="ID of this element in DBRIP">
-##INFO=<ID=MATEID,Number=.,Type=String,Description="ID of mate breakends">
-##INFO=<ID=PARID,Number=1,Type=String,Description="ID of partner breakend">
-##INFO=<ID=EVENT,Number=1,Type=String,Description="ID of event associated to breakend">
-##INFO=<ID=CILEN,Number=2,Type=Integer,Description="Confidence interval around the inserted material between breakends">
-##INFO=<ID=DP,Number=1,Type=Integer,Description="Read Depth of segment containing breakend">
-##INFO=<ID=DPADJ,Number=.,Type=Integer,Description="Read Depth of adjacency">
-##INFO=<ID=CN,Number=1,Type=Integer,Description="Copy number of segment containing breakend">
-##INFO=<ID=CNADJ,Number=.,Type=Integer,Description="Copy number of adjacency">
-##INFO=<ID=CICN,Number=2,Type=Integer,Description="Confidence interval around copy number for the segment">
+##INFO=<ID=MEINFO,Number=.,Type=String,Description="Mobile element info of the form NAME,START,END,POLARITY">
+\end{verbatim}
+\normalsize
+
+If present, the number of entries must be four (4) the number of alt allele (Number=4A).
+$MEINFO$ consists of successive quadruplets of records for each alt allele.
+
+\footnotesize
+\begin{verbatim}
+##INFO=<ID=METRANS,Number=.,Type=String,Description="Mobile element transduction info of the form CHR,START,END,POLARITY">
+\end{verbatim}
+\normalsize
+
+If present, the number of entries must be four (4) the number of alt allele (Number=4A).
+$MEINFO$ consists of successive quadruplets of records for each alt allele.
+
+\footnotesize
+\begin{verbatim}
+##INFO=<ID=DGVID,Number=A,Type=String,Description="ID of this element in Database of Genomic Variation">
+##INFO=<ID=DBVARID,Number=A,Type=String,Description="ID of this element in DBVAR">
+##INFO=<ID=DBRIPID,Number=A,Type=String,Description="ID of this element in DBRIP">
+##INFO=<ID=MATEID,Number=A,Type=String,Description="ID of mate breakends">
+##INFO=<ID=PARID,Number=A,Type=String,Description="ID of partner breakend">
+##INFO=<ID=EVENT,Number=A,Type=String,Description="ID of event associated to breakend">
+##INFO=<ID=CILEN,Number=.,Type=Integer,Description="Confidence interval around the inserted material between breakends">
+\end{verbatim}
+\normalsize
+
+If present, the number of entries must be twice the number of alt allele (Number=2A).
+$CIEND$ consists of successive pairs of records indicate the lower and upper bounds of the $SVLEN$ confidence interval.
+
+\footnotesize
+\begin{verbatim}
+##INFO=<ID=DP,Number=A,Type=Integer,Description="Read Depth of segment containing breakend">
+##INFO=<ID=DPADJ,Number=A,Type=Integer,Description="Read Depth of adjacency">
+##INFO=<ID=CN,Number=A,Type=Integer,Description="Copy number of segment containing breakend">
```

Comment:
As I mentioned in the discussion around https://github.com/samtools/hts-specs/pull/465, I find these descriptions confusing -- what breakend is being referred to exactly? what is the segment that contains it?
 
If we want to use this to represent allele-specific copy number for CNVs, wouldn't it be much clearer to say something like, "allele-specific copy number of the reference segment overlapped by this allele"? What other uses for this field are there? 

If you feel the same way, this might be a decent time to re-write them.

Reply:
I'm actually unclear as to what this field is event meant to be. Is it:

1. The copy number of the segment for a CNV/DEL/DUP record

2. The copy number of the anchoring segment for a breakend

3. The copy number of the breakend itself (e.g. CN 4 -> CN 1 = breakend CN of 3)


It's also unclear as to how we specify ASCN. All done at a sample level?